src/config/config.c

```diff
@@ -1007,6 +1007,12 @@ static void initConfig(struct config *conf)
 	conf->webserver.port.d.s = (char*)"80,[::]:80,443s,[::]:443s";
 	conf->webserver.port.c = validate_stub; // Type-based checking + civetweb syntax checking
 
+	conf->webserver.threads.k = "webserver.threads";
+	conf->webserver.threads.h = "Maximum number of worker threads allowed.\n The Pi-hole web server handles each incoming connection in a separate thread. Therefore, the value of this option is effectively the number of concurrent HTTP connections that can be handled. Any other connections are queued until they can be processed by a unoccupied thread.\n The default value of -1 means that the number of threads is automatically determined by the number of online CPU cores minus 1 (e.g., launching up to 8-1 = 7 threads on 8 cores). A value of -2 means the same automatism but for twice the number of cores (e.g., launching up to 16-1 = 15 threads on 8 cores). A value of 0 (or any other negative number) means that the web server is disabled. Positive values specify the number of threads explicitly. A hard-coded maximum of 64 threads is enforced for this option.\n The total number of threads you see may be lower than the configured value as threads are only created when needed due to incoming connections.";
```

Comment:
I'm confused by 

> (or any other negative number) means that the web server is disabled.

But we set the default to `-1` but the webserver should not be disabled I think?

Reply:
> The default value of -1 means that the number of threads is automatically determined by the number of online CPU cores minus 1 (e.g., launching up to 8-1 = 7 threads on 8 cores).

Reply:
I understand the description:

- `0` = "web server is disabled"

- `-1` = "number of threads is automatically determined by the number of online CPU cores minus 1"
- `-2` = "same as above, but for twice the number of cores"

- `< -3` = "web server is disabled"

... but I understand yubiuser's confusion. This is not very intuitive.

> A hard-coded maximum of 64 threads is enforced for this option

Maybe we should limit the lower bound to `-2` to simplify the description.